.eslintrc

```diff
@@ -63,6 +63,13 @@
         "unused-imports/no-unused-vars": [
           "error",
           { "argsIgnorePattern": "^_" }
+        ],
+        "import/no-extraneous-dependencies": [
+          "error",
+          {
+            "devDependencies": ["**/*.stories.*", "**/.storybook/**/*.*"],
+            "peerDependencies": true
```

Comment:
Could you remove peerDependencies to true? By default, it's already true: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md#importno-extraneous-dependencies-forbid-the-use-of-extraneous-packages